Web/Exercise/02/index.js

```diff
@@ -0,0 +1,29 @@
+let body = document.getElementsByTagName("body")[0];
+let hours = document.getElementById("hr");
+let minutes = document.getElementById("min");
+let seconds = document.getElementById("sec");
+
```

Comment:
`body`, `hours`, `minutes` and `seconds` are never reassigned and so they do not have to be declared using `let`. Can we use `const` instead.